src/SprykerEco/Zed/Unzer/Business/UnzerFacade.php

```diff
@@ -284,6 +285,20 @@ public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredential
         return $this->getFactory()->createUnzerCredentialsCreator()->createUnzerCredentials($unzerCredentialsTransfer);
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsResponseTransfer
+     */
+    public function createUnzerCredentialsAndSetUnzerNotificationUrl(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer
```

Comment:
Sounds like SRP violation

Reply:
We can still remove the "and" from the name of the method, no? What if this needs to be changed? The URL part is removed or something else added? The name won't be correct anymore.

Reply:
Fixed.